### ISSUE TITLE
Supplemental Claims | Confirmation page focus management

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#59546]
## Value Statement
**_As a_** screen reader user submitting a Supplemental Claim form
**_I want_** focus to be placed on an important message 
**_So that_** I know the status of my submission 

---
## Background Context

The code to focus on the `H2` on the confirmation page was in place, but the `H2` is located within a `va-alert` shadow dom, so the focus fails and the page body receives focus. We want the focus to find the `H2` within the `va-alert` shadow dom.

## Tasks
- [ ] Add appropriate focus on confirmation page
- [ ] Add unit test

## Acceptance Criteria
- [ ] Confirmation page initial focus is on the `H2` inside the `va-alert`
- [ ] All tests passing

## Definition of Done

- [ ] Focus on confirmation page is appropriately managed
- [ ] Reviewed and approved by a11y, product and/or design

**Comment by Mottie** [comment]
Fix has been merged in, ready for testing in staging

**Comment by joshkimux** [comment]
Just tested on staging, but I wasn't successful in getting the announcement to read out. Did I test too early? I'll try again first thing next Monday 😄 

https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/ed1dd506-9a4f-4b02-971f-3d4a77814f7c

**Comment by Mottie** [comment]
Hmm, the changes are available in staging... I just tested it in Brave (Chrome-based) and it worked, which browser are you using?

**Comment by joshkimux** [comment]
@Mottie setup was:
- Safari
- MacOS
- VoiceOver

**Comment by joshkimux** [comment]
Safari noooooo 😭 I feel like we've had similar focus issues on other parts of the form before-- specifically with the alerts that appear after editing contact information.

**Comment by Mottie** [comment]
I wonder if it's because the `H2` being focused is _inside_ a `va-alert`

**Comment by joshkimux** [comment]
Is there any way we can test that? That does sound highly likely, but I'm not sure how to fix/get around that 😢

**Comment by Mottie** [comment]
Hmm, nope that's not it... the focus works on the `/notice-of-evidence-needed` page which has an `H3` inside a `va-alert`, and the `H3`s inside the `va-radio` are also getting focus. Ugh.

**Comment by Mottie** [comment]
Ok, I figured it out... let me make a quick change...

**Comment by Mottie** [comment]
Ok, the next fix attempt is merged in and available in staging for testing.
